### ISSUE TITLE
[Test Failure][Accessibility][ESN] The drop-down button of ‘Default package management format’ was truncated 

### ISSUE THREAD

**Issue by v-nancyj** [NuGet/Home#7715]
## Details about Problem

NuGet version: NuGet Client Release-5.0.0-preview2\5.0.0.5782
VS Version: D16.0\28519.51
OS Version: Windows-10-Enetrprise-RS3-ES-ES

## Detailed repro steps 

1.	Launch VS and click menu Tools->NuGet Package Manager->Package Manager Settings->General.
2.	Check the drop-down button of ‘Default package management format’ on right panel.

### Expected

The drop-down button of ‘Default package management format’ showed entirely.

### Actual

The drop-down button was truncated as below screenshot.
![localbug](https://user-images.githubusercontent.com/20632775/51528402-8ef3f080-1e71-11e9-9fd2-74e9299c556d.png)

### Note

The issue only repro on language ESN.

**Comment by nkolev92** [comment]
I can repro the problem. 
This is not a regression and as such it's priority is lower. 

The design of the whole Options dialog is super weird (mix of elements that both resizable and not. 
I'll play around a bit to see what can be done.

**Comment by nkolev92** [comment]
Similar rootcause as https://developercommunity.visualstudio.com/content/problem/833539/typo-in-a-checkbox-caption.html.

**Comment by v-nancyj** [comment]
The issue has been fixed on VS D16.5\29716.60 with implicit NuGet Client Release-5.5.0-Preview2\5.5.0.6382, close it.
